### ISSUE TITLE
Settings -> Accessibility -> Typing assistance -> 'Alert when a key is rejected' cannot be enabled

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [linuxmint/cinnamon#8451]
```
 * Cinnamon version (cinnamon --version) 4.0.9
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 19.1
 * Graphics hardware *and* driver used: Intel onboard graphics, UHD Graphics 620, Mesa 3.0
 * 32 or 64 bit: 64
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
 ```

**Issue**

Settings -> Accessibility -> Typing assistance -> 'Alert when a key is rejected' cannot be enabled

**Steps to reproduce**

Go to that option and try to enable it.

**Expected behaviour**

The user should be able to enable the setting or at least be told why s/he cannot.

**Other information**

I see the problem both (1) on my normal install, on real hardware and (2 on a fairly clean install within a VM.

**Comment by JosephMcc** [comment]
That setting becomes enabled when you turn on Accessibility->Keyboard (required for typing assistance alerts)->Enable audio alerts
